src/engine/experimental/ePBS.md

```diff
@@ -0,0 +1,119 @@
+# Engine API -- ePBS
+
+Engine API changes introduced in ePBS, based on [Cancun](../cancun.md).
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Structures](#structures)
+  - [InclusionListV1](#inclusionlistv1)
+  - [InclusionListSummaryEntryV1](#inclusionlistsummaryentryv1)
+  - [InclusionListStatusV1](#inclusionliststatusv1)
+  - [PayloadAttributesVePBS](#payloadattributesvepbs)
+- [Methods](#methods)
+  - [`engine_getInclusionListV1`](#engine_getinclusionlistv1)
+    - [Request](#request-1)
+    - [Response](#response-1)
+    - [Specification](#specification-1)
+  - [`engine_newInclusionListV1`](#engine_newinclusionlistv1)
+    - [Request](#request-2)
+    - [Response](#response-2)
+    - [Specification](#specification-2)
+  - [`engine_forkchoiceUpdatedVePBS`](#engine_forkchoiceupdatedvepbs)
+    - [Request](#request-3)
+    - [Response](#response-3)
+    - [Specification](#specification-3)
+
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Structures
+
+### InclusionListV1
+This structure maps onto inclusion list object from [{pending EIP}](). The fields are encoded as follows:
+- `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
+- `summary`: `Array of InclusionListSummaryEntryV1` - Array of summary entries. Each object is an `OBJECT` containing the fields of a `InclusionListSummaryEntryV1` structure.
+
+### InclusionListSummaryEntryV1
+This structure maps onto inclusion list summary entry object from [{pending EIP}](). The fields are encoded as follows:
+- `address`: `DATA`, 20 Bytes
+- `gasLimit`: `QUANTITY`, 64 Bits
+
+### InclusionListStatusV1
```

Comment:
I was checking PayloadStatusV1 and where it has a field known as validation error. I believe we can also have it for inclusion list. Basically, if it's a generic error e.g. invalid parent hash, we can return that error as a part of separate error object but if it's an error out of the engine API processing, we can set that error to this field in InclusionListStatusV1. See [this](https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#response) for more reference.

Reply:
> I was checking PayloadStatusV1 and where it has a field known as validation error. I believe we can also have it for inclusion list. Basically, if it's a generic error e.g. invalid parent hash, we can return that error as a part of separate error object but if it's an error out of the engine API processing, we can set that error to this field in InclusionListStatusV1. See [this](https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#response) for more reference.

That's a good idea! Thanks